src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
 
+export interface Figure {
+  shape: 'triangle'| 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
you can access Figure types with this syntax: `Figure['typeName']` - you won't have to retype everything again